### ISSUE TITLE
date/2.4.1 broken on Windows

### ISSUE THREAD

**Issue by acgetchell** [bincrafters/community#999]
### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **date/2.4.1**
  * Operating System+version: **Windows**
  * Compiler+version: **Visual Studio 2015/2017**
  * Conan version: **conan 1.19.1**
  * Python version: **Python 3.7.0**

### Conan profile (output of `conan profile show default` or `conan profile show <profile>` if custom profile is in use)
```
conan profile new cdt --detect
CC and CXX: clang-cl.exe -mrtm, clang-cl.exe -mrtm 
Found clang 8.0
clang>=8, using the major as version
Profile created with detected settings: C:\Users\appveyor\.conan\profiles\cdt
conan profile update env.CONAN_CMAKE_GENERATOR=Ninja cdt
conan profile show cdt
Configuration for profile cdt:
[settings]
os=Windows
os_build=Windows
arch=x86_64
arch_build=x86_64
compiler=clang
compiler.version=8
compiler.libcxx=libstdc++
build_type=Release
[options]
[build_requires]
[env]
CONAN_CMAKE_GENERATOR=Ninja
conan install .. -pr cdt --build=missing
Auto detecting your dev setup to initialize the default profile (C:\Users\appveyor\.conan\profiles\default)
CC and CXX: clang-cl.exe -mrtm, clang-cl.exe -mrtm 
Found clang 8.0
clang>=8, using the major as version
Default settings
	os=Windows
	os_build=Windows
	arch=x86_64
	arch_build=x86_64
	compiler=clang
	compiler.version=8
	compiler.libcxx=libstdc++
	build_type=Release
*** You can change them in C:\Users\appveyor\.conan\profiles\default ***
*** Or override with -s compiler='other' -s ...s***
Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=clang
compiler.libcxx=libstdc++
compiler.version=8
os=Windows
os_build=Windows
[options]
[build_requires]
[env]
CONAN_CMAKE_GENERATOR=Ninja
boost/1.71.0@conan/stable: Not found in local cache, looking in remotes...
boost/1.71.0@conan/stable: Trying with 'conan-center'...
Downloading conanmanifest.txt
..
Downloading conanfile.py
```

### Steps to reproduce (Include if Applicable)

Build project in AppVeyor on Windows:

https://github.com/acgetchell/CDT-plusplus/blob/develop/.appveyor.yml

The date package is broken because it cannot find string_view.

https://ci.appveyor.com/project/acgetchell/cdt-plusplus/build/job/pauvvbq40u3e510i#L24879

This is a bug that has been fixed upstream.

https://github.com/HowardHinnant/date/issues/434

https://github.com/HowardHinnant/date/pull/330#issuecomment-475621811

### Logs (Include/Attach if Applicable)

https://ci.appveyor.com/project/acgetchell/cdt-plusplus

**Comment by SSE4** [comment]
@acgetchell do they plan to make a release with fix?

**Comment by acgetchell** [comment]
Yes, that fix was committed April 5, after the 2.4.1 release.

https://github.com/HowardHinnant/date/commit/8a563041fa65146531249ed557a6839809488521

The latest version of date works with `vcpkg`, for example. I believe if the package is updated to pull that instead of 2.4.1 this issue may be resolved.

https://github.com/HowardHinnant/date/commit/23fa1bb86d24550d99b9b259aa1005d53e5c46ce

**Comment by Croydon** [comment]
This recipe is now in the [Conan Center Index](https://github.com/conan-io/conan-center-index/).

Please migrate to the recipe version there. 
If this is still an issue, please create a new issue in the [Conan Center Index issue tracker](https://github.com/conan-io/conan-center-index/issues).

Thanks!
